<% if resource.errors.any? %>
  <div id="error_explanation">
    <h2>エラーが発生しました。以下ご確認ください。
      <%= I18n.t(count: resource.errors.count,
                 resource: resource.class.model_name.human.downcase)
       %>
    </h2>
    <ul>
      <% resource.errors.full_messages.each do |message| %>
        <li><%= message %></li>
      <% end %>
    </ul>
  </div>
<% end %>
